app/resources/v1/pacbio/pool_resource.rb

```diff
@@ -43,11 +48,22 @@ def library_attributes=(library_parameters)
         @model.library_attributes = library_parameters.map do |library|
```

Comment:
we are using a constant below but not here?

Reply:
I abstracted aliquot attributes because we use it in more than one place as opposed to library attributes. But I will add a library attributes constant for consistency.